src/database/song_seg.py

```diff
@@ -2,19 +2,72 @@
 from database.storinator import Storinator
 
 
+
 class SSegmentation(Storinator):
     def __init__(self):
+        """initialises the database
```

Comment:
I don't think the constructor needs an explanation ;) So Probably just remove it :) And since we don't take any parameters (as self is a base thing), i'd just remove all of this.